tests/e2e/tests/web/critical-path/rdi/add-rdi-instance.e2e.ts

```diff
@@ -140,3 +140,18 @@ test('Verify that button is displayed if user does not enter all mandatory infor
         await browserActions.verifyTooltipContainsText(text, true);
     }
 });
+test('Verify that user can see the Redis Data Integration message on the empty RDI list', async t => {
+    const noInstancesMessage = 'Redis Data Integration (RDI) synchronizes data from your existing database into Redis in near-real-time. We\'ve done the heavy lifting so you can turn slow data into fast data without coding.';
+    // const externalPageLink = 'https://docs.redis.com/rdi-preview/rdi/quickstart/'
+
+    await t.expect(rdiInstancesListPage.emptyRdiList.withText(noInstancesMessage).exists).ok('Empty RDI page message not displayed');
```

Comment:
probably the test doesn't make sence, since we have method addRdi() and if the form doesn't opened all test with adding rdi manually will be failed 
we can just add the first verification to another test